src/nvim/popupmenu.c

```diff
@@ -102,7 +103,7 @@ static void pum_compute_size(void)
 ///                      if false, a new item is selected, but the array
 ///                      is the same
 /// @param cmd_startcol only for cmdline mode: column of completed match
-void pum_display(pumitem_T *array, int size, int selected, bool array_changed, int cmd_startcol)
+void pum_display(pumitem_T *array, int size, int selected, bool array_changed)
```

Comment:
this is a regression. purposeful function arguments are better than ad-hoc global functions to return ad-hoc global values.

Reply:
I'm also a bit hesitant about this. I'll revert this then.